conf/nginx.conf

```
@@ -4,16 +4,30 @@ location / {
     rewrite ^ https://$server_name$request_uri? permanent;
   }
 
-  # Standard nginx configuration
+  gzip off;
   proxy_http_version 1.1;
+  proxy_set_header X-Real-IP $remote_addr;
+  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+  # For Websocket support
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection "upgrade";
   proxy_set_header Host $http_host;
 
+  proxy_redirect off;
+
   proxy_pass http://localhost:__PORT__;
 
   client_max_body_size 16m;
 
+  location ~* \.(css|js)$ {
+    root /home/mobilizon/live/priv/static;
```

```suggestion
    root __FINALPATH__/mobilizon/priv/static;
```

(not 100% sure about the exact syntax but had to do this to fix the blank page (no css/js loading) ... my exact path was : /var/www/mobilizon/mobilizon/priv/static  (note the two /mobilizon !))